policy_tests/run_unit_tests.py

```diff
@@ -22,10 +22,24 @@ def incompatibleReason(test, policy):
         return "PPAC policy must run with heap and userType policies"
     return None
 
-def xfailReason(test, policy):
+def xfailReason(test, policy, runtime):
     if "longjump" in test:
         return "longjump test known to be broken"
 
+    long_tests = [
+        "link_list_works_1",
+        "string_works_1",
+        "bitcount",
+        "fft",
+        "qsort",
+        "rc4",
+        "rsa",
+        "sha",
+        "stringsearch",
+    ]
+    if test in long_tests and "heap" in policy and (runtime == "frtos"):
+        return "long-running tests known to fail with heap policy on frtos"
```

Comment:
suggest adding "due to context switching" or similar, and referencing this commit in https://github.com/draperlaboratory/hope/issues/104 so that we can easily find and remove this code when we get context switching working

Reply:
Sounds good, thanks!